lib/hanami/mailer.rb

```diff
@@ -214,9 +215,24 @@ def initialize(locals = {})
     # @since 0.1.0
     # @api private
     def render(format)
+      layout_template = self.class.layouts(format)
+      if layout_template&.exist?
```

Comment:
If the layout is declared by the mailer, but the file is missing, this should crash.